src/modules/common/renderHtml.js

```diff
@@ -32,18 +33,91 @@ const renderJS = (files: FilesStateType) => `
       item.src = '${URL_PREFIX}' + newSrc
     }
   }
+})();
+
+(function() {
+  var container = document.getElementById('measure-container')
+  function adjustHeight() {
+    if (window.postMessage.length !== 1){
+      setTimeout(adjustHeight, 200);
+    }
+    else {
+      container.setAttribute('style', 'padding: 1px 0;'); // Used for measuring collapsed vertical margins
+      window.ReactNativeWebView.postMessage(container.getBoundingClientRect().height - 2);
+      container.setAttribute('style', '');
+    }
+  }
   
-  document.body.style.display = 'block'
+  window.addEventListener('load', adjustHeight);
+  window.addEventListener('resize', adjustHeight);
 })();
 `
 
-export default (html: string, files: FilesStateType) => {
+export default (html: string, files: FilesStateType, theme: ThemeType) => {
   // language=HTML
   return `
 <html>
-<body style="display: none;">
-${html}
-<script>${renderJS(files)}</script>
+<head>
+  <!-- disables zooming https://stackoverflow.com/questions/44625680/disable-zoom-on-web-view-react-native-->
+  <meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0">
+  <style>
+    html, body {
+        margin: 0;
+        padding: 0;
+        
+        /*font-family: \${theme.fonts.contentFontFamily};*/   
+        font-size: ${theme.fonts.contentFontSize};
+        line-height: ${theme.fonts.contentLineHeight};
+        /*\${props => props.centered && css\`
```

Comment:
Can probably be removed.

Reply:
no probably not; we'll need that once we use webview for categorylists as well